bottlecap/src/main.rs

```diff
@@ -118,21 +119,23 @@ fn main() -> Result<()> {
     // First load the configuration
     let lambda_directory = std::env::var("LAMBDA_TASK_ROOT").unwrap_or("".to_string());
     let config = match config::get_config(Path::new(&lambda_directory)) {
-        Ok(config) => config,
+        Ok(config) => Arc::new(config),
```

Comment:
Using a plain Arc for this makes a lot of sense, since config should never be mutated after this point.